torch/csrc/utils/python_arg_parser.cpp

```diff
@@ -366,6 +366,15 @@ bool is_tensor_list_and_append_overloaded(PyObject* obj, std::vector<py::handle>
   return true;
 }
 
+
+bool is_float_list_and_append_overloaded(PyObject* obj, std::vector<py::handle>* overloaded_args, int argnum, bool throw_error) {
+  // check if obj is a TensorList
+  if (is_tensor_list_and_append_overloaded(obj, overloaded_args, argnum, false)) {
+    return false;
+  }
+  return (PyTuple_Check(obj) || PyList_Check(obj));
+}
```

Comment:
@ezyang, is this a hack?

Reply:
Yes. Besides being a super hack, it probably is also wrong, because I think you are going to append tensor list to the overloaded twice if you do it this way. Why not just implement it the normal way? (You don't need overload handling here because we don't support overloading Scalars).

Reply:
Done